go.mod

```diff
@@ -215,6 +215,8 @@ require (
 	github.com/grafana/grafana/pkg/storage/unified/resource v0.0.0-20250121113133-e747350fee2d // @grafana/grafana-search-and-storage
 )
 
+require github.com/tjhop/slog-gokit v0.1.3 // @grafana/grafana-app-platform-squad
```

Comment:
can you move this into the section with explicit require?  (eg, up three lines and then go-fmt)